src/sqltest/parts/notebook/model/notebookModel.test.ts

```diff
@@ -131,7 +131,7 @@ suite('notebook model', function (): void {
 		mockContentManager.setup(c => c.getNotebookContents(TypeMoq.It.isAny())).returns(() => Promise.resolve(emptyNotebook));
 		notebookManagers[0].contentManager = mockContentManager.object;
 		// When I initialize the model
-		let model = new NotebookModel(defaultModelOptions, undefined, logService, undefined);
+		let model = new NotebookModel(defaultModelOptions, undefined, logService, undefined, undefined);
 		await model.requestModelLoad();
```

Comment:
You should pass in a TestTelemetryService() instance, otherwise this test is going to fail if it ever gets to the point where it tries to send the telemetry call